include/evmc/evmc.h

```diff
@@ -156,6 +156,16 @@ struct evmc_tx_context
     evmc_uint256be chain_id;         /**< The blockchain's ChainID. */
 };
 
+/**
+ * @struct evmc_host_context
+ * Execution context managed by the Host.
+ *
+ * The Host MUST pass the pointer to the execution context to ::evmc_execute_fn.
+ * The VM MUST pass the same pointer back to the Host in every callback function.
+ * The context MUST contain at least the function table defining
```

Comment:
Why is this requirement and how can it be enforced? :)

Reply:
Sorry, this has to be updated. Now:

> Host MAY pass the pointer...
> VM MUST pass the same pointer back...

The last one is not needed any more, as the "context callback interface" has been separated.